### PR TITLE
Add Letter in contact header

### PR DESCRIPTION
**Pull request by skjnldsv**
See #307

### CODE REVIEW

File: templates/contactDetails.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
 <div ng-if="ctrl.contact!==undefined">
 <header class="contactdetails__header" ng-style="{'background-color': (ctrl.contact.uid() | contactColor)}">
 	<img ng-if="ctrl.photo!==undefined" class="contactdetails__logo avatar" data-ng-src="data:image/png;base64,{{ctrl.photo}}" />
+	<div class="contactdetails__logo contact__icon" ng-show="ctrl.contact.photo()===undefined">{{ ctrl.contact.fullName() | firstCharacter }}</div>
```
Comment (Henni) [inline templates/contactDetails.html]: This line should look similar to the line above it:

``` html
<div ng-if="ctrl.photo===undefined" class="contactdetails__logo avatar--missing">{{ ctrl.contact.fullName() | firstCharacter }}</div>
```

**Comment by DeepDiver1975** [comment]
please squash your commits - THX

**Comment by DeepDiver1975** [comment]
just squashed

**Comment by DeepDiver1975** [comment]
tested - works :+1:

**Comment by skjnldsv** [comment]
Thanks deep!

**Comment by skjnldsv** [comment]
@Henni good to you?

**Comment by DeepDiver1975** [comment]
@jancborchardt wants to have a look at this

**Comment by jancborchardt** [comment]
@skjnldsv I see some problems with this, commented on the original issue: https://github.com/owncloud/contacts/issues/307#issuecomment-202898209

**Comment by skjnldsv** [comment]
Closing, see #307

**Pull request closed without merging**